src/main/java/edu/harvard/iq/dataverse/license/LicenseDAO.java

```diff
@@ -23,6 +26,14 @@ public License find(long id) {
     public List<License> findAll() {
         return em.createQuery("SELECT l FROM License l ORDER BY l.position ASC", License.class).getResultList();
     }
+
+    public List<LicenseReorderDto> findLocalizedLicenses(Locale locale) {
+        return em.createQuery("SELECT new edu.harvard.iq.dataverse.license.dto.LicenseReorderDto(l.id,names.text)" +
+                "FROM License l " +
+                "JOIN l.localizedNames names WHERE names.locale = :locale ORDER BY l.position ASC", LicenseReorderDto.class)
+                .setParameter("locale", locale)
+                .getResultList();
```

Comment:
Not so sure about this solution. Now we have two methods of obtaining localized name - that already are slightly different. One here and one in `License.getLocalizedName()`. IMO we should `findAll` and do the conversion License->LicenseReorderDto

Reply:
Hmm in my mind it was due to performance reason (fetch as little info as possible), but I'm fine either way.

Reply:
IMO it is kind of premature optimalisation and it is harder to maintain